pyproject.toml

```diff
@@ -110,6 +110,9 @@ sphinxcontrib-httpdomain = "^1.8.0"
 tabulate = "^0.8.9"
 
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
We are using the `dev-dependencies` notation which will be slowly deprecated in favor of `dev.dependencies`. I am in favor of the new notation but we need to migrate the existing one otherwise it's failing with the following error
```
The Poetry configuration is invalid:
#0 0.589     - Additional properties are not allowed ('group' was unexpected)
```

From the poetry documentation:
```
Poetry will slowly transition away from the dev-dependencies notation which will soon be deprecated, so it’s advised to migrate your existing development dependencies to the new group notation.
```

Reply:
Sorry, I used the wrong version of `poetry` and it used the new format. I'll fix all of this.